src/runtime_src/core/tools/common/XBUtilities.cpp

```diff
@@ -413,7 +413,7 @@ XBUtilities::report_available_devices()
   std::cout << std::endl;
 }
 
-std::pair<const char*, size_t>
+std::pair<std::vector<char>, size_t>

```

Comment:
just return std::vector.  size of vector is same.

Reply:
Oh yes, good point ;)